README.md

```diff
@@ -80,16 +80,17 @@ Alternatively, you could pass the webhook as a configuration option.
+| Option                 | Description                                                                                                                                                                                                                                     | Default                                      |
+| ---------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------- |
+| `notifyOnSuccess`      | Determines if a succesfull release should trigger a slack message to be sent. If `false` this plugin does nothing on success.                                                                                                                   | false                                        |
+| `notifyOnFail`         | Determines if a failed release should trigger a slack message to be sent. If `false` this plugin does nothing on fail.                                                                                                                          | false                                        |
+| `onSuccessTemplate`    | Provides a template for the slack message object on success when `notifyOnSuccess` is `true`. See [templating](#templating).                                                                                                                    | undefined                                    |
+| `onFailTemplate`       | Provides a template for the slack message object on fail when `notifyOnFail` is `true`. See [templating](#templating).                                                                                                                          | undefined                                    |
+| `markdownReleaseNotes` | Pass release notes through markdown to slack formatter before rendering.                                                                                                                                                                        | false                                        |
+| `slackWebhook`         | Slack webhook created when adding app to workspace.                                                                                                                                                                                             | SLACK_WEBHOOK                                |
+| `slackWebhookEnVar`    | Environment variable to use to select the default Slack webhook value.                                                                                                                                                                          | SLACK_WEBHOOK                                |
```

Comment:
Just to clarify so that I understand it correctly. The default value of `slackWebhook` is the *value of* SLACK_WEBHOOK, and for `slackWebhookEnVar` it is actually the string `'SLACK_WEBHOOK'`?

If so, do you think there is a good way to be more clear about this?

Reply:
That is exactly it. I tried to improve the documentation, is it clearer ?

Reply:
I think it was much clearer after your update. I did, however, have some difficulties with understanding the description for `slackWebhookEnVar`. You can see the comment further down 🙂